openshift/recommender.yaml

```diff
@@ -0,0 +1,33 @@
+apiVersion: build.openshift.io/v1
+kind: BuildConfig
+metadata:
+  name: recommender-image
+spec:
+  output:
+    to:
+      kind: ImageStreamTag
+      name: recommender:latest
+  source:
+    images:
+    - from:
+        kind: ImageStreamTag
+        name: shopping-base:latest
```

Comment:
Please also apply memory and cpu usage limits at container level to avoid unnecessary wastage. See for reference: https://docs.openshift.com/container-platform/3.11/admin_guide/limits.html#overview

Reply:
This should be in DeploymentConfig not in BuildConfig. I am considering adding info on how to do that.

Reply:
Added info about setting resource constraints to README.md in the current PR (force pushed with history rewritten).